crates/vector-store/src/engine.rs

```diff
@@ -26,7 +28,7 @@ type GetIndexIdsR = Vec<IndexId>;
 type AddIndexR = anyhow::Result<()>;
 type GetIndexR = Option<(mpsc::Sender<Index>, mpsc::Sender<DbIndex>)>;
 
-pub(crate) enum Engine {
+pub enum Engine {
```

Comment:
```suggestion
pub(crate) enum Engine {
```
Do we need a change in visibility?